x-pack/plugins/uptime/public/components/monitor/ping_list/ping_list.tsx

```diff
@@ -49,6 +49,18 @@ export const formatDuration = (durationMicros: number) => {
   }
   const seconds = (durationMicros / ONE_SECOND_AS_MICROS).toFixed(0);
 
+  if (shortLabel) {
+    if (seconds === '1') {
+      return i18n.translate('xpack.uptime.pingist.durationSecondsColumnFormatting.singular.sec', {
```

Comment:
These two translations look the same to me `defaultMessage: '{seconds} sec'`

Reply:
Displaying full 'Seconds' work in chart axis looks weird, so i adopted short form 'sec' for this

Reply:
That makes sense to me, I'm just wondering if we need two translations since both of the values of the translations are the same? Or is this to provide more flexibility?

Reply:
Speaking of this, I think we should maybe use 's' as outlined by Drew https://docs.google.com/document/d/1r1oW5Sw0g7Y6Z9vuCgdVAIckG8evUFZzoeUt9uu5r60/edit